src/models/document/document-content.ts

```diff
@@ -834,9 +834,10 @@ export const DocumentContentModel = types
     },
     userCopyTiles(tiles: IDragTileItem[], rowInfo: IDropRowInfo) {
       const dropRow = (rowInfo.rowDropIndex != null) ? self.getRowByIndex(rowInfo.rowDropIndex) : undefined;
-      const results = dropRow?.acceptsTileDrops
-                        ? self.copyTilesIntoExistingRow(tiles, rowInfo)
-                        : self.copyTilesIntoNewRows(tiles, rowInfo.rowInsertIndex);
+      const rowDropLocation = rowInfo.rowDropLocation;
+      const results = dropRow?.acceptsTileDrops && ((rowDropLocation === "left") || (rowDropLocation === "right"))
+                      ? self.copyTilesIntoExistingRow(tiles, rowInfo)
+                      : self.copyTilesIntoNewRows(tiles, rowInfo.rowInsertIndex);
```

Comment:
Fix looks good -- from an API standpoint, I was thinking it might make sense to have `acceptsTileDrops` take the `rowInfo` as a parameter and then this code would be:
```suggestion
      const rowDropLocation = rowInfo.rowDropLocation;
      const results = dropRow?.acceptsTileDrops(rowInfo)
                      ? self.copyTilesIntoExistingRow(tiles, rowInfo)
                      : self.copyTilesIntoNewRows(tiles, rowInfo.rowInsertIndex);
```
and the `left`/`right` test could be done in `acceptsTileDrops`. What do you think? (I checked and this is the only client of `acceptsTileDrops`, and many other methods take a `rowInfo` argument so there is precedent.

Reply:
Hmm.... I was under the impression that `acceptsTileDrops` is unrelated to the relative drag position of a tile and is used to prevent drags to the tile header.  That it's more of a way of saying that a row cannot take drops in ANY situation. 
 Either way, the name `acceptsTileDrops` seems wrong if we have it be a function that both specifies if a given row allows drops and if a drop can occur based on specific row info (e.g., the relative drag position of the thing being dropped).  Maybe I just need to sleep on this.

Reply:
> the name `acceptsTileDrops` seems wrong...

Good point -- with my refactor it should be named `acceptsTileDrop`, and then header rows can return false unconditionally while individual rows can return true or false depending on the nature of the drop.

Reply:
The reason I think this API makes sense is that it puts the logic for which branch of the ternary to take in one place (the row) rather than being split between the row and the document content. Maybe in the future there's a kind of row that only accepts drops on the right but not the left.

Reply:
Check out my revisions and see if this is what you were thinking.